### ISSUE TITLE
Pychromecast Daemon?

### ISSUE THREAD

**Issue by ikkysleepy** [balloob/pychromecast#256]
Is there a Daemon for Pychromecast? I am using PI Zero W and running a simple play command with a python script takes ~10 seconds to execute .  I was thinking if there was a deamon that would connect to the Chromecast then I would send pause and play commands to it.

**Comment by balloob** [comment]
Daemon is out of scope for this project.

**Comment by vijrohit** [comment]
There is a lot of delay. <import pychromecast> takes few seconds, then get_chromecasts takes few more seconds and by the time it calls play( ) it's been 8-10 seconds.

I am implementing a script which has a while loop and waiting on listening to a socket. It has already imported the library and has already detected CC on my wifi. 

From my user interface, running on another setup, I send URL in a UDP packet packet. The script gets the URL/play/stop command and runs it immediately.

If you don't like UDP/sockets, you can also use dweet for this purpose. Checkout [dweet.io].

I wish there was a Daemon for pychromecast.

**Comment by ikkysleepy** [comment]
I don't know python, but I started making something similar using FIFO and while loop.  You would start the daemon like so  `pychromecast-daemon.py chromecast_name &`  and then you would echo to the FIFO about 8-10 seconds later, like so:

`echo "play" >> chromecast_name.fifo`

Which does the same thing your script does but using FIFO.  I thought having a pychromecast daemon would handle most corner cases and auto reconnecting such that there wouldn't be much delay after sending a couple of commands then turning off my tv and coming back later to turn on my tv and trying to send new commands.

**Comment by ikkysleepy** [comment]
I had issues with finding my chromecast, so I gave up on this. I wanted to pause and play quickly, so I did this using PHP and CastV2inPHP; I forked the repo and made it such that sending pause or play takes 1-2 seconds if your using an IP and 2-5 seconds if your using a friendly name. There are much less features but now I don't need a daemon anymore and the performance is excellent.  The repo can be found here: https://github.com/ikkysleepy/CastV2inPHP
